xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 9.6.0
+JSONRPC_VERSION 9.6.1
```

Comment:
still wrong

Reply:
No patch bump is correct - see definition of patch https://github.com/xbmc/xbmc/blob/4ad0edc3eba3d5022e20fb8713b1512d0e2210e4/xbmc/interfaces/json-rpc/schema/methods.json#L34
This PR changes xbmc/interfaces/json-rpc/PlayerOperations.cpp the internal implementation of the API.  Schema version will then bump to 10.0.0 for v18RC

If the team wishes to change the _definition_ of the schema version number  for v19 then raise a PR on schema definition etc., meanwhile I am applying the currently correct and documented strategy expected by JSON consumers.